src/m365/commands/setupPresets.ts

```diff
@@ -4,6 +4,7 @@ export const interactivePreset = {
   output: 'text',
   printErrorsAsPlainText: true,
   prompt: true,
+  interactive: true,
```

Comment:
Let's sort properties alphabetically for easier lookup

Reply:
Oh wow, did not think this PR would cause a comment 😆

Reply:
The devil's in the details 😄

Reply:
Using the GitHub app to move a line on my phone ✅